docs/package.json

```diff
@@ -28,7 +28,7 @@
     "@mdx-js/react": "^2.1.0",
     "@moxy/next-compile-node-modules": "^2.1.1",
     "@xstate/inspect": "^0.4.1",
-    "aws-amplify": "^5.0.1",
+    "aws-amplify": "latest",
```

Comment:
To ensure we don't have weird `aws-amplify` install behavior, aligned the dep version here to what our example apps use, which is `latest`

Reply:
Related: https://github.com/aws-amplify/amplify-ui/pull/3497#discussion_r1124064034